Docs.Site/Docs/Reference/Symbols.markdown

```diff
@@ -2,20 +2,59 @@
 
 MyGet symbols support lets consumers of our NuGet packages step through the source code and integrate with Visual Studio and tools like WinDbg.
 
-MyGet comes with its own symbol server that supports pushing symbols as well as consuming symbols and source indexing. Both managed and native assemblies and symbols are supported.
+MyGet comes with its own symbol server that supports pushing .snupkg and legacy .symbol.nupkg symbols, as well as consuming symbols and source indexing. Both managed and native assemblies and symbols are supported.
 
 ## Creating symbols packages
 
-Using the `nuget pack` command, you can create a normal .nupkg and a .symbols.nupkg file with the `-symbols` option. *The symbols package must contain both the .dll and .pdb file, together with the source files*. Check the [NuGet documentation](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages) for more information and an example of a .nuspec file.
+Using the nuget pack command, you can create a normal .nupkg and a symbol file for debugging. On MyGet you can push both your NuGet package and its symbol package to the same feed for easier management. 
+
+### New .snupkg format
+To create your symbol package in the new .snupkg format, you must specify the `SymbolPackageFormat` property.
+* Either add this property to your .csproj file:
+
+    ```PropertyGroup>
+           <IncludeSymbols>true</IncludeSymbols>
+           <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+       </PropertyGroup>```
+
+* Or in the command line:
+    
+    ```dotnet pack MyPackage.csproj -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg```
+   
+    or 
+    
+    ```msbuild MyPackage.csproj /t:pack /p:IncludeSymbols=true /p:SymbolPackageFormat=snupkg```
+
```

Comment:
nuget pack example command is missing

Reply:
@2103962345 I'm confused - isn't the dotnet pack example right under "Or in the command line:"

![image](https://user-images.githubusercontent.com/19432462/75907072-557c0b80-5e0d-11ea-83d0-2d6f29795328.png)

Reply:
@matthcap yes, but dotnet and nuget CLI have different pack commands, but as I see it now, all good